spec/app/domain/util/trackable_log_message_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'Util::TrackableLogMessageClass' do
+  context 'object' do
```

Comment:
You can leave context out here.  You only need it when you have multiple contexts.